ibis/backends/exasol/tests/conftest.py

```diff
@@ -0,0 +1,122 @@
+from __future__ import annotations
+
+import os
+import subprocess
+from typing import TYPE_CHECKING
+
+import ibis
+from ibis.backends.tests.base import (
+    ServiceBackendTest,
+)
+
+if TYPE_CHECKING:
+    from collections.abc import Iterable
+    from pathlib import Path
+    from typing import Any
+
+EXASOL_USER = os.environ.get("IBIS_TEST_EXASOL_USER", "sys")
+EXASOL_PASS = os.environ.get("IBIS_TEST_EXASOL_PASSWORD", "exasol")
+EXASOL_HOST = os.environ.get("IBIS_TEST_EXASOL_HOST", "localhost")
+EXASOL_PORT = int(os.environ.get("IBIS_TEST_EXASOL_PORT", 8563))
+IBIS_TEST_EXASOL_DB = os.environ.get("IBIS_TEST_EXASOL_DATABASE", "EXASOL")
+
+
+class TestConf(ServiceBackendTest):
+    check_dtype = False
+    check_names = False
+    supports_arrays = False
+    supports_arrays_outside_of_select = supports_arrays
+    supports_window_operations = True
+    supports_divide_by_zero = False
+    returned_timestamp_unit = "us"
+    supported_to_timestamp_units = {"s", "ms", "us"}
+    supports_floating_modulus = True
+    native_bool = True
+    supports_structs = False
+    supports_json = False
+    supports_map = False
+    reduction_tolerance = 1e-7
+    stateful = True
+    service_name = "exasol"
+    supports_tpch = False
+    # Question/NOTE: imports do not seem to work yet, if commented in tests will bes skipped
+    # deps = "sqlalchemy", "sqlalchemy-exasol"
+    deps = []
```

Comment:
@cpcloud should we address this in this PR?

Reply:
I see, my bad. I think I did not realize that it is the actual package, rather than the PyPI package name.